nuclear/cmake/Modules/ToolchainLibraryFinder.cmake

```diff
@@ -90,8 +90,8 @@ function(ToolchainLibraryFinder)
     if(PACKAGE_VERSION_BINARY_ARGUMENTS AND PACKAGE_BINARY)
       exec_program(
         ${${PACKAGE_NAME}_BINARY} ARGS
-        ${PACKAGE_VERSION_BINARY_ARGUMENTS} OUTPUT_VARIABLE
```

Comment:
How is this relevant to this PR?

Reply:
Why are you commenting on merge commits? This is the merge in from master